### PR TITLE
ITEA-8-create-configuration-parser

### PR DESCRIPTION
**Pull request by RomanPryima**
First variant of parser. Still unknown the format of data it will be using for.

### CODE REVIEW

File: config_parser/config_reader.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"""
+Module for getting value of certain parameter in certain section in cfg. file
+"""
+import ConfigParser
+
+
+def get_setting(cfg_file, section,  parameter):
```
Comment (UkrainianEngineer) [inline config_parser/config_reader.py]: too many spaces before `parameter`

File: config_parser/config_reader.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    """
+    cfg_file - name or path and name of cfg file for parsing
```
Comment (UkrainianEngineer) [inline config_parser/config_reader.py]: It would be great to follow:
http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Comment (RomanPryima) [reply config_parser/config_reader.py]: Fixed. As I was able to understand how to.

File: config_parser/config_reader.py [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
 import ConfigParser
 
 
 def get_setting(cfg_file, section,  parameter):
     """
-    cfg_file - name or path and name of cfg file for parsing
-    section - name of section of parameter
-    parameter - name of parameter of value has to be received
+    Args:
+        cfg_file - name or path and name of cfg file for parsing
```
Comment (UkrainianEngineer) [inline config_parser/config_reader.py]: It would be great to add type of variable.

File: config_parser/config_reader.py [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
+        section - name of section of parameter
+        parameter - name of parameter of value has to be received
+    Returns:
+        The return value. The value of requested parameter in requested
```
Comment (UkrainianEngineer) [inline config_parser/config_reader.py]: It's awesome! 😎 
`Return return value` - Captain Obvious, it's you? 😂 

Well, it should describe type of return value and reason of it.

File: config_parser/config_reader.py (hunk 3 above)
Comment (RomanPryima) [reply config_parser/config_reader.py]: As you know in cfg file value can be any type. So function can bring different types of data.

File: config_parser/config_reader.py (hunk 4 above)
Comment (RomanPryima) [reply config_parser/config_reader.py]: What I can write here if it returns exactly that?

File: config_parser/config_reader.py (hunk 1 above)
Comment (RomanPryima) [reply config_parser/config_reader.py]: fixed

File: config_parser/config_reader.py (hunk 4 above)
Comment (UkrainianEngineer) [reply config_parser/config_reader.py]: What is `that`? It's not clear from your sentence "Returns the return value."

Comment (RomanPryima) [reply config_parser/config_reader.py]: corrected

File: config_parser/config_reader.py (hunk 3 above)
Comment (RomanPryima) [reply config_parser/config_reader.py]: corrected

File: config_parser/config_reader.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def get_setting(cfg_file, section,  parameter):
+    """
+    Args:
+        cfg_file (str) - name or path and name of cfg file for parsing
+        section (str) - name of section of parameter
+        parameter (str) - name of parameter of value has to be received
+    Returns:
+        string: The value of requested parameter in requested
```
Comment (UkrainianEngineer) [inline config_parser/config_reader.py]: `str` not `string`.

**Review by UkrainianEngineer — CHANGES_REQUESTED** [review]

File: config_parser/config_reader.py (hunk 5 above)
Comment (RomanPryima) [reply config_parser/config_reader.py]: corrected

File: config_parser/config_reader.py (hunk 1 above)
Comment (UkrainianEngineer) [reply config_parser/config_reader.py]: Are you sure that it's fixed? I don't think so.

**Review by UkrainianEngineer — CHANGES_REQUESTED** [review]

File: config_parser/config_reader.py (hunk 1 above)
Comment (RomanPryima) [reply config_parser/config_reader.py]: Now it surely done.

**Comment by UkrainianEngineer** [comment]
LGTM, :+1:

**Review by UkrainianEngineer — APPROVED** [review]

**Review by kate-ka — APPROVED** [review]
👍

**Comment by Sasha1152** [comment]
:+1:

**Review by Sasha1152 — APPROVED** [review]
:+1:  :+1:  :+1:

**Pull request merged**